### PR TITLE
Add Working Group Workload Overview

### PR DESCRIPTION
**Pull request by 4n4nd**
Add a new Working Group that will work on creating a overview dashboard for operate first workloads

### CODE REVIEW

File: wg-workload-overview/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-operations-leads
```
Comment (HumairAK) [inline wg-workload-overview/OWNERS:4]: did you mean to write `wg-workload-overview-leads:`?

Comment (4n4nd) [reply wg-workload-overview/OWNERS:4]: I did

**Review by schwesig — APPROVED** [review]
/lgtm
/approved

**Review by schwesig — APPROVED** [review]
/lgtm
/approved

**Pull request merged**